pkg/mount/mountutil.go

```diff
@@ -185,3 +269,23 @@ func (m *DiskMountUtil) getMountName(mountLine string) (DeviceMountAttr, bool) {
 	}
 	return mountAttr, isValid
 }
+
+func getCmdlineFile() string {
+	cmdlineFile := "/host/proc/cmdline"
```

Comment:
nit: Would be good to define `/host/proc/cmdline` and `/proc/cmdline` as a constant and then use it here.

Reply:
done